osu.Game/Screens/Edit/Components/EditorSidebarSection.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Localisation;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osuTK;
+
+namespace osu.Game.Screens.Edit.Components
+{
+    public class EditorSidebarSection : Container
+    {
+        protected override Container<Drawable> Content { get; }
+
+        public EditorSidebarSection(LocalisableString sectionName)
+        {
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+
+            InternalChild = new FillFlowContainer
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Direction = FillDirection.Vertical,
+                Children = new Drawable[]
+                {
+                    new SectionHeader(sectionName),
+                    Content = new Container
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                    },
+                }
+            };
+        }
+
+        public class SectionHeader : CompositeDrawable
+        {
+            private readonly LocalisableString text;
+
+            public SectionHeader(LocalisableString text)
+            {
+                this.text = text;
+
+                Margin = new MarginPadding { Vertical = 10, Horizontal = 5 };
+
+                AutoSizeAxes = Axes.Both;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OverlayColourProvider colourProvider)
+            {
+                InternalChildren = new Drawable[]
+                {
+                    new OsuSpriteText
+                    {
+                        Text = text,
+                        Font = OsuFont.Default.With(size: 16, weight: FontWeight.SemiBold),
+                    },
```

Comment:
This implementation is not super resilient against text potentially not fitting in a single line which may become a problem especially with localisation. Maybe fine to ignore until it actually becomes a problem, but shouldn't take too much work to address now either if you decide it's worth it.

Reply:
Fair point, I'll make it wrap (thought I'd probably hope we don't do this as it won't look amazing).

Reply:
Probably best to address if it's a simple change (seems like just using TFC).